Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/RecordNavPage.xaml

```diff
@@ -6,31 +6,39 @@
       xmlns:local="clr-namespace:Ginger.BusinessFlowsLibNew.AddActionMenu"
+      xmlns:Ginger="clr-namespace:Ginger"
       mc:Ignorable="d" 
-      Title="RecordNavAction">
+      Title="RecordNavAction" >
 
     <Grid>
         <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="300*"/>
+            <ColumnDefinition Width="400*"/>
         </Grid.ColumnDefinitions>
         <Grid.RowDefinitions>
-            <RowDefinition Height="30" />
+            <RowDefinition Height="45" />
+            <RowDefinition Height="40" />
             <RowDefinition Height="Auto" />
-        </Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>        
+        
+        <userControl:UCWindowsGrid x:Name="xWinGridUC" Margin="0, 5, 0, 0" Grid.Row="0" AddSwitchActionRequired="True"  />
 
-        <userControl:UCWindowsGrid x:Name="xWinGridUC" Grid.Row="0" AddSwitchActionRequired="False" />
+        <usercontrols:ucButton Grid.Row="1" x:Name="xRecordingButton" Margin="3,0,0,0" ButtonType="RoundTextAndImageButton" ButtonText="Start Recording" ButtonImageType="Play" 
+                               HorizontalAlignment="Left" VerticalAlignment="Center" ToolTip="Start Recording" ButtonFontImageSize="15" ButtonImageWidth="10" 
+                               ButtonImageHeight="15" Click="RecordingButton_Click" ButtonStyle="{StaticResource $RoundTextAndImageButtonStyle_Execution}"/>
 
-        <Button x:Name="RecordingButton" Click="RecordingButton_Click"  IsEnabled="true" ToolTip="Start Recording" HorizontalAlignment="Left" VerticalAlignment="Top" Width="Auto" Margin="10" UseLayoutRounding="True" Background="DeepSkyBlue" Grid.Row="1">
-            <Grid>
-                <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="40" />
-                    <ColumnDefinition Width="Auto" />
-                </Grid.ColumnDefinitions>
+        <Label x:Name="xStartAgentMessage"  Grid.Row="2" Content="Please start the Agent, currently it is not running" Style="{StaticResource @InputFieldLabelStyle}" FontWeight="Bold" VerticalAlignment="Center" HorizontalAlignment="Left" FontSize="13"/>
 
-                <Ellipse Name="startRecord" Height="25" Width="25" Fill="Red" Grid.Column="0" Margin="0,10" VerticalAlignment="Center" Visibility="Visible"/>
-                <Rectangle Name="stopRecord" Height="25" Width="25" Fill="Red" Grid.Column="0" Margin="0,10" VerticalAlignment="Center" Visibility="Collapsed" />
-                <Label Name="lblRecording" Grid.Column="1" Content="Start Recording" FontSize="15" FontWeight="Bold" Foreground="White" />
-            </Grid>
-        </Button>
+        <Label x:Name="xAgentNotSupportRecording"  Grid.Row="3"  Content="Recording not supported with selected Agent" Style="{StaticResource @InputFieldLabelStyle}" FontWeight="Bold" VerticalAlignment="Center" HorizontalAlignment="Left" FontSize="13"/>
```

Comment:
This is added, since we are not changing the state of window, if user visits the "RecordNav" page and changes the businessflow then in the next businessflow that window is kept open so it is used to show the message to user

Reply:
if user shift BusinessFlow we should go back to main page and not still stuck on current open page